ui/pages/permissions-connect/snaps/snap-install/snap-install.js

```diff
@@ -187,10 +192,12 @@ export default function SnapInstall({
                 snapName={snapName}
                 permissions={requestState.permissions || {}}
                 connections={requestState.connections || {}}
+                onShowAllPermissions={onShowAllPermissionsHandler}
               />
             </Box>
-            {isScrollable && !isScrolledToBottom ? (
-              <Box className="snap-install__scroll-button-area">
+
+            <Box className="snap-install__scroll-button-area">
+              {isScrollable && !hasScrolledToBottom && !showAllPermissions ? (
```

Comment:
Why do we not have the scroll button on screen if we aren't showing all permissions? Even if we have filtering the list could be long